Tutorial.md

```diff
@@ -77,22 +76,14 @@ zcat sorted.vcf.gz \
 **Note:** `svtools lmerge` will return variant lines for SECONDARY breakends in addition to merging variants.
 This will sometimes cause the merged VCF to have more variant lines than the input.
 
-### (Optional) Remove variants detected by alignment to the EBV (Epstein-Barr Virus) contig
-If you used the recommended reference sequence, this step is unnecessary. However, if your reference contains a contig representing EBV then you may wish to remove SVs involved with this sequence.
-```
-zcat merged.vcf.gz \
-| vawk --header '{if($0 !~ /NC_007605/) print $0}' \
-| bgzip -c > merged.no_EBV.vcf.gz
-```
-
-### Use `svtools genotype` to force genotypes for variant positions discovered in other samples
-`svtools genotype` will calculate a genotype for each sample at the variant positions in the merged.no_EBV.vcf.gz file.
+### Use `svtools genotype` to genotype all samples for all variants present in the merged set for variant positions discovered in other samples
```

Comment:
"for variant positions discovered in other samples" seems to be be redundant here.